website/index.html

```diff
@@ -211,10 +211,25 @@ <h4 id="bonus">Bonus</h4>
             <p>Add a static import for <code>ButterKnife.findById</code> and enjoy even more fun.</p>
 
             <h3 id="download">Download</h3>
-            <h4>Gradle</h4>
-            <pre class="prettyprint">compile 'com.jakewharton:butterknife:<span class="version"><em>(insert latest version)</em></span>'
-apt 'com.jakewharton:butterknife-compiler:<span class="version"><em>(insert latest version)</em></span>'</pre>
+            Configure your project-level build.gradle to use at least version 2.2.0 of the Android Gradle Plugin:
+            <h4>Project-level build.gradle</h4>
+            <pre class="prettyprint">
+buildscript {
+    ...
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'  // 2.2.0 or higher
+    }
+}
+            </pre>
 
+            Then add the Butter Knife dependencies to the module-level build.gradle:
+            <h4>Module-level build.gradle</h4>
+            <pre class="prettyprint">
+dependencies {
+  compile 'com.jakewharton:butterknife:8.4.0</span>'
+  annotationProcessor 'com.jakewharton:butterknife-compiler:8.4.0</em></span>'
```

Comment:
Both of these should use `<span class="version"><em>(latest version)</em></span>` for the version.